### ISSUE TITLE
Source for board editor?

### ISSUE THREAD

**Issue by llimllib** [ornicar/chessground#21]
Hi,

You list a board editor demo [which is sweet](http://en.lichess.org/editor/rnbqkbnr/pp1ppppp/2p5/8/4P3/8/PPPP1PPP/RNBQKBNR_w_KQkq_-_1_2), but the source for it [is minified](http://lichess1.org/assets/compiled/lichess.editor.min.js?v=757). Does that source exist somewhere in a readable format?

**Comment by llimllib** [comment]
OK, I'll [answer my own question](https://github.com/ornicar/lila/tree/master/ui/editor), fair enough.

Still the source should be linked. Should I open up a PR to add links to the source to the README?

**Comment by llimllib** [comment]
OK, I'm going to reopen this, because how the heck do you build the actual HTML page?

**Comment by llimllib** [comment]
Trying to build the editor js, I got a bunch of uninstalled package errors:

```
$ gulp --gulpfile ui/editor/gulpfile.js dev
[10:58:53] Working directory changed to ~/code/lila/ui/editor
module.js:338
    throw err;
          ^
Error: Cannot find module 'isarray'
...
```

Which I solved by `npm install isarray` (I had to do it for about 10 packages...) but eventually I get to:

`Error: Cannot find module './lib/_stream_readable.js'`

Which I don't know how to solve. Any idea what's going on there?

(I'm not trying to run another lichess, btw! I'd like to make a chess analysis app and use chessground as its chessboard because it's much better than the alternatives. Let me know if I'm stepping on your toes)
